src/main/java/eu/tsystems/mms/tic/testerra/plugins/xray/mapper/xray/XrayTestExecutionImport.java

```diff
@@ -324,19 +331,8 @@ public void addEvidence(Evidence evidence) {
         private Status status;
         private List<Step> steps;
 
-        public TestRun() {
-        }
-
-        public TestRun(JiraIssue issue) {
-            this(issue.getKey());
-            this.testInfo = new Info();
-            this.testInfo.setDescription(issue.getDescription());
-            this.testInfo.setSummary(issue.getSummary());
-            this.testInfo.setLabels(issue.getLabels());
-            this.testInfo.setDefinition(issue.getSummary());
-            this.testInfo.setType(TestType.AutomatedGeneric);
-            this.testInfo.setProjectKey(issue.getProject().getKey());
-        }
+//        public TestRun() {
```

Comment:
Should either be removed or a heading comment added, what to do with this code.

Reply:
My mistake. Is still needed by Jackson parser.